internal/fwserver/server_planresourcechange_test.go

```diff
@@ -379,6 +400,51 @@ func TestServerPlanResourceChange(t *testing.T) {
 		},
 	}
 
+	testSchemaDefault := schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"test_optional_bool": schema.BoolAttribute{
+				Optional: true,
+				Default:  booldefault.StaticValue(true),
```

Comment:
Nit: Not for this PR per-se, but this is code we should raise a validation error diagnostic. Defaults _must_ be declared with at least `Computed: true` to signal to Terraform that the provider may return its own value for the attribute. Otherwise, Terraform will return its own error if the provider ever sets the proposed new state value with a null configuration value.

Reply:
Let's have a chat about this one.